src/components/Header.tsx

```diff
@@ -43,34 +151,52 @@ export const Header: React.FC<HeaderProps> = ({ navigation }) => {
         },
       )}
     >
-      <div className="mr-6 lg:hidden">
-        <MobileNavigation navigation={navigation} />
-      </div>
       <div className="relative flex flex-grow basis-0 items-center">
         <Link href="/">
-          <a className="block w-10 overflow-hidden lg:w-auto">
-            <span className="sr-only">Home page</span>
+          <a className="block w-auto overflow-hidden lg:w-auto">
             <Logo variant={theme.isDarkTheme ? 'dark' : 'light'} />
           </a>
         </Link>
       </div>
-      {!isHomePage && (
-        <div className="-my-5 mr-6 sm:mr-8 md:mr-0">
-          <Search />
+      <div
+        className={clsx('relative ml-auto hidden items-center lg:flex', {
+          'md:flex': isHomePage,
+        })}
+      >
+        <nav className="text-sm font-semibold leading-6 text-slate-700 dark:text-slate-200">
+          <ul className="flex space-x-8">
+            <NavItems />
+          </ul>
+        </nav>
+        <div className="ml-6 flex items-center border-l border-slate-200 pl-6 dark:border-slate-800">
+          <ThemeSelector panelClassName="mt-8" />
```

Comment:
`panelClassname` is not a valid syntax in our context.

![CleanShot 2022-10-29 at 13 09 22@2x](https://user-images.githubusercontent.com/11677026/198818913-adfe3735-7f07-4da6-87cc-d9b79c5bf5c2.png)

Reply:
* Mode icon, logo fixed
* `panelClassname` removed